src/components/signup-view/signup-view.jsx

```diff
@@ -1,55 +1,122 @@
 import { React, useState } from "react";
+import FloatingLabel from 'react-bootstrap/FloatingLabel';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
 
-export const SignupView = () => {
+export const SignupView = ({onLoggedIn}) => {
 
-    const handleSubmit = (event) => {};
     const [username, setUsername] = useState("");
     const [password, setPassword] = useState("");
     const [email, setEmail] = useState("");
-    const [birthday, setBirthday] = useState("");
+    const [birthdate, setBirthdate] = useState("");
+    const Backend_API = "https://guarded-peak-19726.herokuapp.com";
+    
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        
+            const data = {
+                username: username,
+                password: password,
+                email: email,
+                birthdate: birthdate
+              };
+              
+              fetch(`${Backend_API}/users`, {
+                method: "POST",
+                body: JSON.stringify(data),
+                headers: {
+                  "Content-Type": "application/json"
+                }
+              }).then(res.json).then(
```

Comment:
This line should be updated to:
```suggestion
              }).then(res => res.json).then(
```
Personally, I would discourage you from doing this(login after signup). Because this only creates an over headed with what are supposed cover in the course. I would encourage you to only take step further after you are comfortable with the exercise, and you feel like you need an extra challenge.